manticore/platforms/evm.py

```diff
@@ -1754,11 +1753,10 @@ def LOG(self, address, size, *topics):
     def read_buffer(self, offset, size):
         if size:
             self._allocate(offset+size)
-
         data = []
         for i in xrange(size):
-            data.append(Operators.CHR(self._load(offset+i)))
-
+            data.append(self._load(offset+i))
+        data = map(Operators.CHR, data)
```

Comment:
As we eventually want to go to Python 3 world (https://github.com/trailofbits/manticore/issues/45) this should be `list(map(...))`.

The `map` returns a generator-like object in Py3 instead of a `list`.

Reply:
Making that a generator object should be okay; it's iterated/transformed below. On a more structural note, the above can be expressed as:

```
data = [Operators.CHR(self._load(i)) for i in range(offset, offset+size)]
```